bd-logger/src/consumer.rs

```diff
@@ -39,7 +41,10 @@ struct Flags {
   // The duration to wait before uploading an incomplete batch. The batch will be uploaded
   // at this point regardless of log activity.
   batch_deadline_watch: Watch<u32, bd_runtime::runtime::log_upload::BatchDeadlineFlag>,
-  // The interval at which to check the buffer for new records.
+
+  // The lookback window for the flush buffer uploads.
```

Comment:
nit: can you say what unit this is? (It would be kind of cool if we had some type of helper which auto converted into a Duration, an idea for some other time.)